CONTRIBUTING.md

```diff
@@ -72,6 +76,19 @@ make test-wasm-browser
 make test-all
 ```
 
+### Test Issues
+Some of the tests will open a fairly large number of open files. The default allowance
+for open files on most operating systems is 64, which will cause these tests to fail. The
```

Comment:
Mac OS defaults to 256 and AFAIK no one else has run into any issues before. Maybe we could rephrase this as something like:

> The default maximum number of open files for some operating systems is too low for the tests to work properly. If you see an error along the lines of "Too many open files", you may need to increase this limit.